common/static/coffee/src/discussion/utils.coffee

```diff
@@ -113,7 +113,7 @@ class @DiscussionUtil
       alertDiv = $("<div class='modal' role='alertdialog' id='discussion-alert' aria-describedby='discussion-alert-message'/>").css("display", "none")
       alertDiv.html(
         "<div class='inner-wrapper discussion-alert-wrapper'>" +
-        "  <button class='close-modal dismiss' aria-hidden='true'><i class='icon fa fa-times'></i></button>" +
+        "  <button class='close-modal dismiss' aria-hidden='true'><span class='icon fa fa-times' aria-hidden='true'></span></button>" +
```

Comment:
Sorry @clrux, I'm not clear about what you are recommending. For instance, if the button has a title, does the icon then not need sr text? Should aria-hidden be removed from the button? Can you provide the desired HTML for this case so it is clear? Thanks.